crypto/rsa/rsa_lib.c

```diff
@@ -326,6 +326,8 @@ uint16_t ossl_ifc_ffc_compute_security_bits(int n)
         return 176;
     case 8192:
         return 200;
+    case 15360:
```

Comment:
Hmm, thinking about this more should we somehow make 256 to be the value returned also for the <15360 bit keys for which the formula would return >256 strength?

I.E., something like `if (n < 15360 && formula_result >256) return 256;`

Also the NIST FIPS 140 IG special-cases also the 7680 bits as having 192 bit strength. Should that be included as well?

Reply:
So why do we ever return something > 256?

In theory this function returns the security strength of the modulus, and so can in theory can return something larger than 256. But in practice anything claiming to be larger than 256 doesn't make sense.

Reply:
Read the discussion in #15421...

Reply:
I've added a special case for 7680 bits.  This will require reapproval.

I'm less sure about the >256 change.

Reply:
To avoid going larger than 256: `if (n > 13913) return 256;` is the ideal short circuit path.  That the largest value where the formula returns less than 256.

Reply:
I think the current change is ok, but I wouldn't mind this > 13913 change either.